yang/frr-pim.yang

```diff
@@ -0,0 +1,253 @@
+module frr-pim {
+  yang-version "1.1";
+  namespace "http://frrouting.org/yang/pim";
+
+  prefix frr-pim;
+
+  import frr-interface {
+    prefix frr-interface;
+  }
+
+  import frr-routing {
+    prefix "frr-rt";
+  }
+
+  import ietf-inet-types {
+    prefix "inet";
+  }
+
+  organization
+    "Free Range Routing";
+
+  contact
+    "FRR Users List:       <mailto:frog@lists.frrouting.org>
+     FRR Development List: <mailto:dev@lists.frrouting.org>";
+
+  description
+    "The module defines a collection of YANG definitions common for
+     PIM (Protocol Independent Multicast) model.";
+
+  revision 2017-03-09 {
+    description
+      "Initial revision.";
+    reference
+      "RFC XXXX: A YANG Data Model for PIM";
+  }
+
+  /*
+   * Groupings
+   */
+
+  grouping sm_common-config-attributes {
```

Comment:
All of the items here are configured on a per instance / VRF basis, and aren't specific to PIM-SM; what was the reasoning for placing them all in an SM-specific grouping?

Reply:
As explained followed IETF initially, we will move grouping to global and use them directly under VRF.